packages/manager/apps/telecom/src/components/telecom/telephony/scheduler/actions/bankHolidays/telephony-scheduler-bank-holidays.html

```diff
@@ -42,15 +42,13 @@
                     <div class="btn-group-vertical w-100"
                          role="group"
                          data-translate-attr="{ 'aria-label': 'telephony_scheduler_bank_holidays_year_aria_label' }">
-                        <button class="btn btn-default"
-                                type="button"
-                                data-ng-class="{ active: SchedulerHolidaysCtrl.model.year === year }"
-                                data-ng-repeat="year in SchedulerHolidaysCtrl.yearList track by $index"
-                                data-ng-model="SchedulerHolidaysCtrl.model.year"
-                                data-uib-btn-radio="'{{ year }}'">
-                            <span data-ng-bind="year"></span>
-                            <span data-ng-bind="'(' + SchedulerHolidaysCtrl.filterEvents(year).length + ')'"></span>
-                        </button>
+                        <oui-radio-toggle-group class="year-list" data-model="SchedulerHolidaysCtrl.model.year">
```

Comment:
```suggestion
                        <oui-radio-toggle-group data-model="SchedulerHolidaysCtrl.model.year">
```
I don't see the class being used originally

Reply:
The UI originally had the `uib-btn-radio` displayed as a column (meaning the buttons, which denotes the years like 2018, 2019 etc are displayed one below the other).  But the `oui-radio-toggle` component does not support this vertical orientation. This class `year-list` has been added to change the orientation of the buttons to vertical and to handle some css changes, that have been explained in the comment above.

Reply:
Maybe instead of adding custom styles we could see if it makes sense to add that as an option to `oui-radio-toggle-group`.

Reply:
Point taken. A new ticket is created for this: UK-125. Once done, the manager can be updated with the new option